[Docs] Enhance Notification Center Documentation (#14348)
meshery/meshery: docs/pages/guides/infrastructure-management/events-management.md

---
layout: default
title: Managing Events with Notification Center
abstract: Meshery tracks operations performed on your infrastructure and workloads, and provides notification of environment issues, application conflicts with infrastructure configuration, policy violations, and so on.
permalink: guides/infrastructure-management/notification-management
redirect_from: guides/events-management
type: guides
category: infrastructure
language: en
---

> Meshery continuously tracks operations performed on your infrastructure and workloads. While running, Meshery Server generates a variety of events reflecting activities inside your Kubernetes and Cloud-Native clusters, as well as within Meshery itself.

Reviewer: ```suggestion
Meshery continuously tracks activities affecting your infrastructure and workloads. Meshery generates a variety of events, reflecting activities affecting the design and operation of your infrastructure whether those activities are directly or indirectly performed through Meshery operations.
```

> ### Introduction

Reviewer: ```suggestion
### What is the Notification Center?
```

More descriptive of what the user will learn, so that they can decide whether or not to read the section.

An introduction might describe how the process of managing infrastructure causes events to be generated, and so on.

The Notification Center is a dedicated panel in Meshery’s UI that helps you monitor, understand, and respond to events across your system. It acts as a central place where you can see important updates related to your infrastructure, workloads, and Meshery’s internal operations.

![Find Notification Center]({{ site.baseurl }}/assets/img/notification-center/find-notification-center.gif)

> ### Event Descriptions

Reviewer: ```suggestion
### Types of Notifications

Given the variety of activities that occur through the process of managing infrastructure, notifications have been grouped into categories so that you can readily understand what a notification means and what do to about any particular type you have received.
```

</a>

### Types of Notifications

Given the variety of activities that occur through the process of managing infrastructure, notifications have been grouped into categories so that you can readily understand what a notification means and what do to about any particular type you have received.

Each notification in Meshery includes a clear summary of what occurred in your system. Notifications vary in format depending on the event type, but typically include:

- Action performed (e.g., saved a design, ran a dry run)
- Affected components (e.g., deployments, MeshSync, events)
- Validation results or errors (e.g., invalid values, missing fields)
- Relationship updates (e.g., how components are linked)
- Links to the related design or further details

You can mark notifications as read or unread to stay organized and focused. Meshery highlights critical, actionable events in red, helping you quickly spot and respond to urgent issues.

<a href="{{ site.baseurl }}/assets/img/notification-center/description.png">
  <img src="{{ site.baseurl }}/assets/img/notification-center/description.png" alt="Event Descriptions" style="width: 600px; height: auto; margin-top: 10px;" />
</a>

> > 🔗 For more technical details, see the [Contributor Reference](#).

Author: ```suggestion
> 🔗 For more technical details, see the [Contributor Reference](https://docs.meshery.io/project/contributing/contributing-ui-notification-center).
```
